gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjurePlugin.java

```diff
@@ -147,20 +155,36 @@ private static void setupConjureJavaProject(
         if (JAVA_PROJECT_SUFFIXES.stream()
                 .anyMatch(suffix -> project.findProject(project.getName() + suffix) != null)) {
             Configuration conjureJavaConfig = project.getConfigurations().maybeCreate(CONJURE_JAVA);
-            File conjureJavaDir = new File(project.getBuildDir(), CONJURE_JAVA);
+            conjureJavaConfig.getAttributes().attribute(ARTIFACT_TYPE, "tgz");
```

Comment:
I'm not sure you need to apply the attribute on the config if you're going to use an artifact view

Reply:
I think the input artifact type must be tgz for any of this to work, so this should constrain the configuration to only tgz artifacts if I understand correctly. The docs leave a lot to be desired though, and mention that `artifactType` is "special".